INSTALL.md

```diff
@@ -1,5 +1,35 @@
+## Operating system support
+
+Dangerzone can run on a varieties of Operating systems, and have automated tests for a subset of them.
+This section explains what is the support you should expect from us.
+
+You can find general support information in this table, and more details in the following sections.
+
+(Unless specified, the architecture is AMD64)
+
+| Distribution | Supported versions        | Automated tests        | Manual QA         |
+| ------------ | ------------------------- | ---------------------- | ----------------- |
+| Windows      | 3 last releases           | Last release only      | Last version only |
+| macOS intel  | 3 last releases           | Last release only      | Last release only |
+| macOS ARM    | 3 last releases           | Last release only      | Last release only |
```

Comment:
For macOS with Intel CPU, our automated tests take place on macOS 13. The same applies to our manual tests. For macOS ARM (I'd refer to it as "Apple Silicon" though), the manual tests take place on macOS 13 as well.

Since the available devices can change from time to time though, I'd prefer a checkmark here than a specific version.

Reply:
I've added some more information about how we run our tests, and a link to the github runners, see c0a4f388b36669d0fdbe3726b2f2cc47e7cdca28

Reply:
The thing is that our test devices are on an earlier version, so it's not accurate :-/. So, either we should be more accurate (and keep this section in mind when updating our devices), or just add a tick there, to show that we manually test Dangerzone on macOS.

Reply:
Ah, gotcha!

For macOS, I'm always testing on the latest macOS release (on my `m1` machine), but this is not true for intel mac and for windows.

I've updated everything with a checkbox, and squashed the commits.

Reply:
Makes sense, thanks!